apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/FissClaimResponseTransformerV2.java

```diff
@@ -0,0 +1,153 @@
+package gov.cms.bfd.server.war.r4.providers.preadj;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import com.google.common.collect.ImmutableMap;
+import com.newrelic.api.agent.Trace;
+import gov.cms.bfd.model.rda.PreAdjFissClaim;
+import gov.cms.bfd.sharedutils.exceptions.BadCodeMonkeyException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import org.hl7.fhir.r4.model.ClaimResponse;
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.Extension;
+import org.hl7.fhir.r4.model.Identifier;
+import org.hl7.fhir.r4.model.Meta;
+import org.hl7.fhir.r4.model.Reference;
+
+/** Transforms FISS/MCS instances into FHIR {@link ClaimResponse} resources. */
+public class FissClaimResponseTransformerV2 {
+
+  private static final String METRIC_NAME =
+      MetricRegistry.name(FissClaimResponseTransformerV2.class.getSimpleName(), "transform");
+
+  private static final Map<Character, String> STATUS_TEXT;
+
+  static {
+    STATUS_TEXT =
+        ImmutableMap.<Character, String>builder()
+            .put('a', "Active")
+            .put('s', "Suspend")
+            .put('p', "Paid")
+            .put('d', "Denied")
+            .put('i', "Inactive")
+            .put('r', "Reject")
+            .put('t', "RTP")
+            .put('m', "Move")
+            .build();
+  }
+
+  private static final Map<Character, ClaimResponse.RemittanceOutcome> STATUS_TO_OUTCOME;
+
+  static {
+    STATUS_TO_OUTCOME =
+        ImmutableMap.<Character, ClaimResponse.RemittanceOutcome>builder()
+            .put(' ', ClaimResponse.RemittanceOutcome.QUEUED)
+            .put('a', ClaimResponse.RemittanceOutcome.QUEUED)
+            .put('s', ClaimResponse.RemittanceOutcome.PARTIAL)
+            .put('p', ClaimResponse.RemittanceOutcome.COMPLETE)
+            .put('d', ClaimResponse.RemittanceOutcome.ERROR)
```

Comment:
I discussed this with Nicole, and for the time being we're going to remap the `d, r and t` states to "COMPLETE".  I've updated the table in the confluence page to reflect this.  Sorry for the last minute change